mantle/system/ns/enter.go

```diff
@@ -51,7 +52,7 @@ func Enter(ns netns.NsHandle) (func() error, error) {
 
 // NsCreate returns a handle to a new network namespace.
 // NsEnter must be used to safely enter and exit the new namespace.
-func Create() (netns.NsHandle, error) {
+func Create() (_ netns.NsHandle, err error) {
 	runtime.LockOSThread()
 	defer runtime.UnlockOSThread()
```

Comment:
How about:
```suggestion
func Create() (netns.NsHandle, error) {
      var err error
	runtime.LockOSThread()
	defer runtime.UnlockOSThread()
```
Otherwise we're mixing unnamed and named returns vars.